<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiCrudSymfony2</class>
 <widget class="QMainWindow" name="MultiCrudSymfony2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiCrudSymfony2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabPasos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Preparación</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Intérprete PHP (C:/php/php.exe)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="interpretePhpLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="interpretePhpPushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Examinar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directorio de las Entidades (C:/www/symfony2/src/Acme/DemoBundle/Entity)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directorio del Proyecto (C:/www/symfony2)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre del Bundle (AcmeDemoBundle)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="proyectoLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EntidadesLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="BundleLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bundle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="proyectoPushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Examinar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EntidadesPushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Examinar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejecución</string>
     </attribute>
     <widget class="QPushButton" name="pasoMapeoPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Mapeo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pasoNamespacePushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Namespace</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pasoEntitiesPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Entities</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pasoCrudPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Crud</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionMapeo"/>
    <addaction name="actionNamespace"/>
    <addaction name="actionEntities"/>
    <addaction name="actionCrud"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionMapeo">
   <property name="text">
    <string>Mapeo</string>
   </property>
  </action>
  <action name="actionNamespace">
   <property name="text">
    <string>Namespace</string>
   </property>
  </action>
  <action name="actionEntities">
   <property name="text">
    <string>Entities</string>
   </property>
  </action>
  <action name="actionCrud">
   <property name="text">
    <string>Crud</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
